test/e2e/placement_test.go

```diff
@@ -44,6 +46,18 @@ var _ = ginkgo.Describe("Placement", func() {
 	})
 
 	ginkgo.AfterEach(func() {
+		ginkgo.By("Delete managedclusterset")
+		err = clusterClient.ClusterV1alpha1().ManagedClusterSets().Delete(context.Background(), clusterSet1Name, metav1.DeleteOptions{})
```

Comment:
I think you would want to ignore the notfound error

Reply:
updated, remove the error check.